web-lib-funcs.pl

```diff
@@ -958,7 +958,7 @@ sub PrintHeader
 	print "X-Frame-Options: SAMEORIGIN\n";
 	}
 if (!$gconfig{'no_content_security_policy'}) {
-	print "Content-Security-Policy: script-src 'self' 'unsafe-inline' 'unsafe-eval'; frame-src 'self'; child-src 'self'\n";
+	print "Content-Security-Policy: script-src 'self' 'unsafe-inline' 'unsafe-eval'; frame-src 'self'; child-src 'self' blob:\n";
```

Comment:
What does "blob" do here exactly?

Reply:
This is needed to use inline js workers when tree data is fetched, but it might be a security issue and I found there should be a more strict way to allow it only for the specific script, so let me check. I'll report as soon as I find time to work on it again.